test/e2e/clusters/clusters.go

```diff
@@ -0,0 +1,256 @@
+package clusters
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	k3dCluster "github.com/rancher/k3d/v5/cmd/cluster"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	kindCluster "sigs.k8s.io/kind/pkg/cluster"
+	kindcmd "sigs.k8s.io/kind/pkg/cmd"
+)
+
+// DistroToUse is an "enum" for helping determine which k8s distro to run the tests on.
+type DistroToUse int
+
+const (
+	// DistroUnknown is the "enum" representation for when we don't know what distro the user wants.
```

Comment:
we already have constants used in the codebase to designate distros that look like they are compatible with this code, [distro.go](https://github.com/defenseunicorns/zarf/blob/master/cli/internal/k8s/distro.go#L10).  Seems like it would be cleaner to use that vs buildout an iota enum + map.

Reply:
Different need. Here we are getting user input about what cluster they want. There we are determining what cluster Zarf is being run against.

Reply:
Different usage of the same data in my mind.  One is saying "this is what this cluster is" the other is saying "this is the cluster I want".  But it's still the same data in the end.  Just don't see the need to add logic with same values elsewhere--but I won't lose sleep over this going into the merge.

Reply:
I'd like to leave it for now. It's already gone through a couple of iterations. I'll make an issue in the backlog to revisit it.

Reply:
Created #411